src/components/Steps/index.js

```diff
@@ -44,57 +67,78 @@ class Steps extends React.Component {
             },
         ];
 
+        if (this.state.map) {
```

Comment:
As we talked it should be merged into steps array and we should render next step only if previous ones are filled correctly. To be implemented in seperate issue.